accord-core/src/main/java/accord/coordinate/MaybeRecover.java

```diff
@@ -108,7 +108,7 @@ protected void onDone(Success success, Throwable fail)
                     if (hasMadeProgress(full))
                     {
                         if (full.durability.isDurable())
-                            node.send(topologies.forEpoch(txnId.epoch()).forKey(route.homeKey()).nodes, to -> new InformDurable(to, topologies, route, txnId, full.executeAtIfKnown(), full.durability));
+                            InformDurable.informHome(node, topologies, txnId, route, full.executeAtIfKnown());
```

Comment:
commented in `InformDurable.informHome` but durability "could" be `Universal` yet this method hard codes `Majority`

Reply:
I'll parameterise, but as mentioned in other location if unspecified the lowest level of _durable_ durability is reasonable to assume (should perhaps remove `Local` durability to make it clearer this is the lowest _durable_ durability).